pkg/util/unixsock.go

```diff
@@ -38,11 +38,12 @@ func Reader(r io.Reader, cmd string) []string {
 				fulllines = nil
 			}
 
-			if !strings.HasPrefix(resp[len(resp)-1], IstgtHeader) &&
+			if len(resp) != 0 && !strings.HasPrefix(resp[len(resp)-1], IstgtHeader) &&
```

Comment:
Can we break this check to a bool function e.g.
```go
func isResponseEOL(resp []strings) bool {
...
}
```

Then write the exhaustive UT.

Reply:
@AmitKumarDas , i am writing the unit test cases for the modified code according to the above comment. if it is urgent, can we merge this change and I will raise another PR by tomorrow ?